### DISCUSSION TITLE
pyload error on setup

### DISCUSSION THREAD

**Discussion by maxdanc** [Freetz-NG/freetz-ng#773] [Allgemein]
hi, there is a problem with the pyload setup .. when you run rc.pyload setup this error is returned
```
"
root@fritz:/var/mod/root# rc.pyload setup
Traceback (most recent call last):
  File "/opt/pyLoad/pyLoadCore.py", line 49, in <module>
    from module.network.RequestFactory import RequestFactory
  File "/opt/pyLoad/module/network/RequestFactory.py", line 22, in <module>
    from Browser import Browser
  File "/opt/pyLoad/module/network/Browser.py", line 6, in <module>
    from HTTPRequest import HTTPRequest
  File "/opt/pyLoad/module/network/HTTPRequest.py", line 21, in <module>
    import pycurl
ImportError: File not found
"
```
initially I thought it was a pycurl error but I created several images with the curl versions (instead of 7.43.0) 7.43.0.1 - 7.43.0.5 - 7.44.1 - 7.45.2 and the error is always present.
i think it's a problem that the library is not present or reachable but i don't know what/how to check .. any good soul have any suggestions or can fix the problem?
Thank you.

**Comment by fda77** [comment]
Looks like configuration problem of pyload

**Comment by Er4twXz** [comment]
I run into the same issue -- any solution ?

**Comment by fda77** [comment]
Maybe you need a newer version of https://github.com/Freetz-NG/freetz-ng/tree/master/make/pkgs/python-pycurl ?
But not the latest latest 7.45.2 as it requires Python 3 https://pypi.org/project/pycurl/#Requirements

**Reply by fda77** [reply]
For sure you could add Python 3 to Freetz

**Comment by maxdanc** [comment]
Hi fda77, can you explain how to install python3? I can't find any reference in the menuconfig tool.. is it necessary to install it from the shell on the router? if yes, how? I can't find the ipkg/opkg installers. txs.

**Reply by fda77** [reply]
It doesnt exist
